HW1/P4/P4 bfs.py

```diff
@@ -0,0 +1,50 @@
+#I discussed the solution for this problem with Avery Faller.
+
+# import findspark
+# findspark.init()
+import pyspark
+import numpy as np
+
+
+def copartitioned(RDD1, RDD2):
+    "check if two RDDs are copartitioned"
+    return RDD1.partitioner == RDD2.partitioner
+
+
+def filterer(i):
+    def filt(x):
+        return x==itr_acc.value
+    return filt
+
+def get_first_non_empty_cogroup(x2):
```

Comment:
A more descriptive variable name than x2 would make this more readible.  Also some doc strings on the help functions would be nice